pkg/ncproxy/ncproxygrpc/v1/networkconfigproxy.pb.go

```diff
@@ -358,7 +358,6 @@ var xxx_messageInfo_CreateNetworkRequest proto.InternalMessageInfo
 
 type Network struct {
 	// Types that are valid to be assigned to Settings:
-	//
```

Comment:
Do we know why this file changed?

Reply:
This might be a function of the fact that my local dev box uses golang 1.20 but the CI is currently on 1.19. I have a PR for updating to 1.20 here https://github.com/microsoft/hcsshim/pull/1801 but I need to investigate the test runs.

Reply:
Can you elaborate on why this is happening? Is gofmt running on this file and producing a different result between 1.19 and 1.20 or something?

Reply:
This seems like another case of a generated file changing without the contents its generated from being touched.

Reply:
I'll double check. I pulled this file from main so that it would match the existing code when I changed it back, but I might have missed something.

Reply:
Fixed